premailer/premailer.py

```diff
@@ -563,8 +566,11 @@ def transform(self, html=None, pretty_print=True, **kwargs):
             # attributes, with their single-character equivalents.
             if self.preserve_handlebar_syntax:
                 out = re.sub(
-                    r'="%7B%7B(.+?)%7D%7D"',
-                    lambda match: '="{{' + unescape(unquote(match.groups()[0])) + '}}"',
+                    r'="([^"]*)%7B%7B(.+?)%7D%7D([^"]*?)"',
```

Comment:
To also support lxml version 5 (https://github.com/peterbe/premailer/issues/297) , i propose the following change.



```suggestion
                    r'="([^"]*)(?:%7B%7B|{{)(.+?)(?:%7D%7D|}})([^"]*?)"',
```